contracts/core/modules/RebalancingSetCTokenExchangeIssuanceModule.sol

```diff
@@ -86,7 +87,7 @@ contract RebalancingSetCTokenExchangeIssuanceModule is
 
     /**
      * Withdraw any base Set components to the user from the contract. If component is a supported cToken,
```

Comment:
Unclear to me what purpose of this function is if we aren't unwrapping the cToken. Doesn't that already exist in the inherited contract?